target/cheri-common/cheri-compressed-cap/cheri_compressed_cap_common.h

```diff
@@ -838,8 +840,13 @@ static inline bool _cc_N(setbounds_impl)(_cc_cap_t* cap, _cc_addr_t req_base, _c
     _cc_length_t new_top = new_cap._cr_top;
 
     if (exact) {
+#ifndef CC_IS_MORELLO
```

Comment:
This diff is unrelated to the commit it's in (and is Morello-specific so shouldn't be in this PR?)

Reply:
I meant "touches target-independent files" in the PR title, but I agree we should split commits.

Reply:
Merged into the previous commit.